### DISCUSSION TITLE
End time of event

### DISCUSSION THREAD

**Discussion by kevin-vanb** [liedekef/events-made-easy#472] [General]
Hello,

I recently was informed by a visitor that the end time of the event was not visible. We organise meetings in the weekends and that comes with start and end hour, but only the start hour is visible.

Is there a setting that I can change to show it to visitors?

Kind regards
Kevin

**Comment by liedekef** [comment]
What you show in the event listing or per single event is up to you. EME gives a basis, but the rest is configurable in the "format" settings (or templates). See the doc here on placeholders for events: https://www.e-dynamics.be/wordpress/event-formatting/ ==> by default EME uses "#_STARTDATE - #_STARTTIME" for the single event format, but feel free (and you should) change that using the placeholders in the link above.

**Comment by kevin-vanb** [comment]
Thanks for the tip Franky :-)
